src/features/users/users.repository.ts

```diff
@@ -46,36 +46,84 @@ export const transferOwnershipToUser = (
       },
     })
 
-    await Promise.all(
-      usersToMigrate.map(({ id }) =>
-        Promise.all([
-          prismaSession.groupAdministrator.updateMany({
-            where: {
-              userId: id,
-            },
-            data: {
-              userId,
-            },
-          }),
-          prismaSession.groupParticipant.updateMany({
-            where: {
-              userId: id,
-            },
-            data: {
-              userId,
-            },
-          }),
-          prismaSession.simulation.updateMany({
-            where: {
-              userId: id,
-            },
-            data: {
-              userId,
-            },
-          }),
-        ])
-      )
-    )
+    const userIds = usersToMigrate.map(({ id }) => id)
+    const [newUserGroups, oldUsersGroups] = await Promise.all([
```

Comment:
```suggestion
    const [newUserGroupIds, oldUsersGroups] = await Promise.all([
```
Qu'en penses-tu ? Pour clarifier ce que contiennent chaque variables - respectivement des ids et des objets `group`